### PR TITLE
release: 0.9.0

### PR DESCRIPTION
**Pull request by raelga**
- File based OLM catalog https://github.com/3scale-ops/prometheus-exporter-operator/pull/56
- ARM64 and AMD64 support https://github.com/3scale-ops/prometheus-exporter-operator/pull/56
- GH Actions release workflows for the operator, bundle and catalog https://github.com/3scale-ops/prometheus-exporter-operator/pull/58
- Support for tolerations https://github.com/3scale-ops/prometheus-exporter-operator/pull/59
- Updates all exporters ti the latest compatible version https://github.com/3scale-ops/prometheus-exporter-operator/issues/57
- Improves Makefile target names and help output https://github.com/3scale-ops/prometheus-exporter-operator/pull/68/commits/0906f55043c7068d09df88795aca9c6e7e815aa4

/kind release
/priority important-soon
/assign

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -213,129 +251,96 @@ ifneq ($(origin CATALOG_BASE_IMG), undefined)
...
-
-catalog-publish: catalog-add-bundle catalog-push catalog-retag-latest ## Builds and pushes the catalog image
+bundle-publish: prepare-release bundle-push ## Publish new bundle.
 
-release-publish: container-push bundle-publish catalog-publish ## Publish a new stable release (operator, catalog and bundle)
+catalog-publish: catalog-add-bundle catalog-push catalog-push-latest ## Build and push the catalog image.
 
-get-new-release:
-	@hack/new-release.sh v$(VERSION)
+release-publish: container-push bundle-publish catalog-publish ## Publish a new stable release (operator, catalog and bundle).
```
Comment (slopezz) [inline Makefile:322]: Accordign to alpha release docs https://github.com/3scale-ops/prometheus-exporter-operator/blob/main/docs/release.md#alpha

This target need to be executed manually for alpha releases, however the target help comment says it is only for new "stable" releases

I guess this target is shared for both alpha and stable right? so target help comment should be changed?

File: Makefile [hunk 2]
```diff
@@ -213,129 +251,96 @@ ifneq ($(origin CATALOG_BASE_IMG), undefined)
...
 	if echo $(VERSION) | grep -q 'alpha'; \
 		then $(MAKE) prepare-alpha-release; \
 		else $(MAKE) prepare-stable-release; \
 	fi
 
-prepare-alpha-release: bundle ## Prepare alpha release
+prepare-alpha-release: bundle # Prepare alpha release.
 
-prepare-stable-release: bundle ## Prepare stable release
+prepare-stable-release: bundle # Prepare stable release.
```
[the remark this answers is not in the document]
Comment (raelga) [reply Makefile:315]: That's actually the reason, those targets are internal, superseeded by `prepare-release` that will execute alpha or stable accordingly.

File: Makefile (hunk 1 above)
Comment (raelga) [reply Makefile:322]:
```suggestion
release-publish: container-push bundle-publish catalog-publish ## Publish a new release (operator, catalog and bundle).
```

File: Makefile (hunk 2 above)
[the remark this answers is not in the document]
Comment (slopezz) [reply Makefile:315]: Ah OK got it, so please update release docs with new target names

[the remark this answers is not in the document]
Comment (raelga) [reply Makefile:315]: fixed in https://github.com/3scale-ops/prometheus-exporter-operator/pull/68/commits/bfa537670a72a0514e15328ead9e5e59d6658f24

**Review by slopezz — APPROVED** [review]

**Review by roivaz — APPROVED** [review]

**Pull request merged**